Resources/About Contents.html

```diff
@@ -54,66 +59,69 @@ <h5>VLCFORIOSVERSION<br /><br />MOBILEVLCKITVERSION</h5>
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/ndfred/xcode-tools/blob/master/update_strings.py">update_strings.py</a><br />
 Copyright &copy; 2010 Frédéric Sagnes - <a href="#mit">MIT License</a><br />
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/fkuehne/upnpx">upnpx</a><br />
-Copyright &copy; 2011-2013 Bruno Keymolen, Jonathan Guan, Mujtaba Hassanpur, &copy; 2013-2017 Felix Paul K&uuml;hne - <a href="#bsd3clause">3-clause BSD License</a><br />
+Copyright &copy; 2011-2013 Bruno Keymolen, Jonathan Guan, Mujtaba Hassanpur, &copy; 2013-2019 Felix Paul K&uuml;hne - <a href="#bsd3clause">3-clause BSD License</a><br />
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/fkuehne/WhiteRaccoon">WhiteRaccoon</a><br />
 Copyright &copy; 2011 Valentin Radu, &copy; 2013 Felix Paul K&uuml;hne - <a href="#mit">MIT License</a><br />
 <br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/karlvr/XKKeychain">XKKeychain</a><br />
+Copyright &copy; 2014-2016 Karl von Randow - <a href="#mit">MIT License</a><br />
+<br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/xmartlabs/XLPagerTabStrip">XLPagerTabStrip</a><br />
+Copyright &copy; 2014-2019 xmartlabs - <a href="#mit">MIT License</a><br />
+<br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://aomedia.googlesource.com/aom">aom</a><br />
+Copyright &copy; 2016-2019, Alliance for Open Media - <a href="#bsd2clause">BSD-2</a><br />
+<br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="http://code.google.com/p/libass/">libass</a><br />
-Copyright &copy; 2006-2017 Grigori Goronzy and contributors - <a href="#isc">ISC License</a><br />
+Copyright &copy; 2006-2019 Grigori Goronzy and contributors - <a href="#isc">ISC License</a><br />
 <br />
-&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://libav.org">FFmpeg</a><br />
-Copyright &copy; 2000-2017 the FFmpeg developers - <a href="#lgpl2">LGPLv2.1 or later</a><br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://code.videolan.org/videolan/dav1d">dAV1d</a><br />
+Copyright &copy; 2018-2019, VideoLAN and dav1d authors - <a href="#bsd2clause">BSD</a><br />
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/videolabs/libdsm">liBDSM</a>
-<br />Copyright &copy; 2014-2017 VideoLabs SAS and contributors - <a href="#lgpl2">LGPLv2.1 or later</a><br />
+<br />Copyright &copy; 2014-2019 VideoLabs SAS and contributors - <a href="#lgpl2">LGPLv2.1 or later</a><br />
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.videolan.org/developers/libdvbpsi.html">libdvbpsi</a><br />
 Copyright &copy; 2001-2017 VideoLAN and authors - <a href="#lgpl2">LGPLv2.1 or later</a><br />
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="http://matroska.org">libebml</a><br />
 Copyright &copy; 2002-2017 Steve Lhomme - <a href="#lgpl2">LGPLv2.1 or later</a><br />
 <br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://libav.org">FFmpeg</a><br />
+Copyright &copy; 2000-2019 the FFmpeg developers - <a href="#lgpl2">LGPLv2.1 or later</a><br />
+<br />
```

Comment:
This should link to FFmpeg.

Reply:
will change, I just moved your code though ;)